### PR TITLE
support single azure connection for same channel

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: mod_azure_transcribe/azure_transcribe_glue.cpp [hunk 1]
```diff
@@ -524,6 +583,23 @@ extern "C" {
 
 		frame.data = data;
 		frame.buflen = SWITCH_RECOMMENDED_BUFFER_SIZE;
+		if (cb->is_keep_alive) {
+			if (++cb->frame_count * 20 /*ms*/ / 1000 >= AZURE_KEEP_ALIVE_INTERVAL_SECOND) {
```
Comment (davehorton) [inline mod_azure_transcribe/azure_transcribe_glue.cpp:587]: tell me more about what we are doing here?

**Review by davehorton — CHANGES_REQUESTED** [review]
this whole feature needs to be enabled by an env var in freeswitch, same as Deepgram

File: mod_azure_transcribe/azure_transcribe_glue.cpp (hunk 1 above)
Comment (xquanluu) [reply mod_azure_transcribe/azure_transcribe_glue.cpp:587]: I want to keep printing the log for testing and later we can debug that the bug is still alive and azure session is still alive.

**Comment by xquanluu** [comment]
There is env variable to enable this feature: AZURE_SPEECH_USE_SINGLE_CONNECTION

I have tested this on jambonz.me and thing is working as my expected, long duration transcribe is working, disable gather for 5 mins and re-gather still working.

File: mod_azure_transcribe/azure_transcribe_glue.cpp (hunk 1 above)
Comment (xquanluu) [reply mod_azure_transcribe/azure_transcribe_glue.cpp]: After successfully debug, I removed this code.

**Review by davehorton — APPROVED** [review]

**Pull request merged**